itest/mint_batch_stress_test.go

```diff
@@ -30,29 +31,109 @@ var (
 )
 
 func testMintBatch100StressTest(t *harnessTest) {
-	mintBatchStressTest(t, 100, defaultWaitTimeout)
+	timeout := defaultWaitTimeout
+
+	// If we create a second tapd instance and sync the universe state,
+	// the synced tree should match the source tree.
+	bob := setupTapdHarness(
+		t.t, t, t.lndHarness.Bob, nil,
+	)
+	defer func() {
+		require.NoError(t.t, bob.stop(!*noDelete))
```

Comment:
Maybe we should make the `noDelete` package level flag part of the `harnessTest` so we don't need to provide any special build/test flags when running the code externally?